manifests/u/UpNotePte/Ltd/UpNote/942/UpNotePte.Ltd.UpNote.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: UpNotePte.Ltd.UpNote
+PackageVersion: "942"

```

```suggestion
PackageVersion: 9.10.0
```
Please update file path to match this change.